### DISCUSSION TITLE
Spooky missions

### DISCUSSION THREAD

**Discussion by YmirtheGamer** [FakeFishGames/Barotrauma#8135] [Ideas and suggestions]
Soo i have been playing barotauma with a group of friends and there are some things i thought would fit the game well. Some might not agree but this is my opinion.

Soo my idea is: Make a new kind of mission (or same objective missions as before but somehow convey the idea that there might be something haunting there, or that the route is cursed etc) then during said mission some off putting things could happen, for example:

-Random chatter in the radio (perhaps someone dead is speaking to you)
-Lights going out suddenly
-Noises of aliens but there isnt anyone there.
-Ghost pings on sonar
-Flickering lights while using spotlights outside of sub
-hallucinations?
-sounds of water or fire
-while controlling the sub or viewing any other monitors you might see something else instead
-doors opening on their own

Although some of these happen when you are going crazy, these events would happen to every player at the same time

Not sure if anyone else would like these or if they fit in the game but i got the idea from Darkwood and the night time in that game, when random events might occur and its really spooky :D

**Comment by MaaiG** [comment]
> -Random chatter in the radio (perhaps someone dead is speaking to you)

Lovely! Just like everything else.
👁️ https://github.com/Regalis11/Barotrauma/discussions/7461#discussioncomment-1983915
